spec.html

```diff
@@ -224,17 +224,25 @@ <h1>ShadowRealmImportValue ( _specifierString_, _exportNameString_, _callerRealm
 				1. If _hasOwn_ is *false*, throw a *TypeError* exception.
 				1. Let _value_ be ? Get(_exports_, _string_).
 				1. Let _realm_ be _f_.[[Realm]].
-				1. Return ? GetWrappedValue(_realm_, _value_).
+				1. Return ? GetTransferrableValue(_realm_, _value_).
 			</emu-alg>
 		</emu-clause>
 
-		<emu-clause id="sec-getwrappedvalue" aoid="GetWrappedValue">
-			<h1>GetWrappedValue ( _callerRealm_, _value_ )</h1>
+		<emu-clause id="sec-gettransferrablevalue" aoid="GetTransferrableValue">
+			<h1>GetTransferrableValue ( _receiverRealm_, _value_ )</h1>
 			<emu-alg>
-				1. Assert: _callerRealm_ is a Realm Record.
+				1. Assert: _receiverRealm_ is a Realm Record.
 				1. If Type(_value_) is Object, then
-				  1. If IsCallable(_value_) is *false*, throw a TypeError exception.
-				  1. Return ! WrappedFunctionCreate(_callerRealm_, _value_).
+					1. If IsCallable(_value_) is *true*,
+						1. Return ! WrappedFunctionCreate(_receiverRealm_, _value_).
+					1. If _value_ has an [[ArrayBufferData]] internal slot, then
+						1. If IsSharedArrayBuffer(_value_) is *true*, then
```

Comment:
```suggestion
					1. If _value_ has an [[ArrayBufferData]] internal slot, and if IsSharedArrayBuffer(_value_) is *true*, then
```

@caridy PTAL and feel free to commit the suggestion if that looks better.